roles/ceph-common/tasks/facts.yml

```diff
@@ -17,6 +17,9 @@
 - set_fact:
     init_system={{ init_system.content | b64decode }}
 
+- set_fact:
+    use_systemd={{ init_system == 'systemd\n' }}
```

Comment:
Odd, I'm not seeing a newline when running this on a 14.04 box w/ init:

https://gist.github.com/mattt416/54194ce68183da2bc8f42d6c87885314

Also, you could use the replace filter to strip off the newline, but as you can see in my gist I had to replace('\\n','') for some reason.